_layouts/post.html

```diff
@@ -91,12 +91,11 @@ <h5>{{site.messages.share_this_post}}:</h5>
 						d="M12 20a8 8 0 1 0 0-16 8 8 0 0 0 0 16zm0 2C6.477 22 2 17.523 2 12S6.477 2 12 2s10 4.477 10 10-4.477 10-10 10zm-3.11-8.83l-2.498-.779c-.54-.165-.543-.537.121-.804l9.733-3.76c.565-.23.885.061.702.79l-1.657 7.82c-.116.557-.451.69-.916.433l-2.551-1.888-1.189 1.148c-.122.118-.221.219-.409.244-.187.026-.341-.03-.454-.34l-.87-2.871-.012.008z" />
 				</svg>
 			</a>
-			</a>
 			<a class="mastodon-link btn btn-secondary"
-                href="https://mastodon.social/share?text={{ page.title | uri_escape }}&url={{ page.url | absolute_url | uri_escape }}" rel="noopener" target="_blank">
-                <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" width="24" height="24" fill="currentColor">
-                    <path d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2zm1.844 15h-3.688c-.387 0-.732-.147-.98-.395-.25-.25-.395-.595-.395-.98V9.375h2.25v5.25h3V9.375h2.25V16c0 .387-.147.732-.395.98-.25.25-.595.395-.98.395z" />
-        </svg>
+                		href="https://toot.kytta.dev?text={{ page.title | uri_escape }}&url={{ page.url | absolute_url | uri_escape }}" rel="noopener" target="_blank">
```

Comment:
Seems like it doesn't take `url` query param, might be worth improving it like so:
```suggestion
                		href="https://toot.kytta.dev?text={{ page.title | uri_escape }}%20{{ page.url | absolute_url | uri_escape }}" rel="noopener" target="_blank">
```